sparse/unit_test/Test_Sparse_Transpose.hpp

```diff
@@ -55,7 +55,7 @@ void testTranspose(int numRows, int numCols, bool doValues) {
   using rowmap_t    = typename crsMat_t::row_map_type::non_const_type;
   using entries_t   = typename crsMat_t::index_type::non_const_type;
   using values_t    = typename crsMat_t::values_type::non_const_type;
-  size_type nnz     = 10 * numRows;
+  size_type nnz     = (numRows * numCols) ? 0 : 10 * numRows;
```

Comment:
Probably need to switch the two result values and maybe make the first boolean expression more explicit?
```suggestion
  size_type nnz     = (numRows * numCols > 0) ? 10 * numRows : 0;
```